shell/tabry_fish.fish

```diff
@@ -0,0 +1,108 @@
+
+set -x _tabry_rs_import_path "$_tabry_rs_import_path:~/.local/share/tabry"
+
+if not set -q _tabry_rs_executable
+  set script_dir (dirname (status filename))
+  set -g _tabry_rs_executable "$script_dir/target/debug/tabry"
+end
+
+function tabry_completion_init
+  # Three separate complete calls here are needed to support files.
+  # One provides completions in the case where tabry offers _only_ arguments,
+  # another provides completions where tabry offers _only_ files,
+  # the third handles the case where tabry offers both files and arguments
+  set cmd $argv[1]
+  complete -f -c "$cmd" -n "__fish_tabry_check_only_args" -a "(__fish_tabry_completions)"
+  complete -c "$cmd" -n "__fish_tabry_check_only_file"
+  complete -c "$cmd" -n "__fish_tabry_check_args_and_file" -a "(__fish_tabry_completions)"
+end
+
+# return true if tabry only reports commands
+function __fish_tabry_internal_invoke
+  set SCRIPT (status --current-filename)
+  set SCRIPT_DIR (dirname $SCRIPT)
+
+  # -C "Get cursor position"
+  set cursor_position (commandline -C)
+  set cmd (commandline)
+
+  # todo: here
```

Comment:
Is this accidentally left in or is there something to do here?
Actually, it looks like this is supposed to be tabry_rs_import_path (singular). Sorry for that change, I think I did it so the old Ruby tabry wouldn't interfere with the Rust tabry